test/test_util.py

```diff
@@ -436,41 +445,43 @@ def test_netloc(self, url, expected_netloc):
     ]
 
     @pytest.mark.parametrize("url, expected_url", url_vulnerabilities)
-    def test_url_vulnerabilities(self, url, expected_url):
+    def test_url_vulnerabilities(
+        self, url: str, expected_url: Union["Literal[False]", Url]
+    ) -> None:
         if expected_url is False:
             with pytest.raises(LocationParseError):
                 parse_url(url)
         else:
             assert parse_url(url) == expected_url
 
-    def test_parse_url_bytes_type_error(self):
+    def test_parse_url_bytes_type_error(self) -> None:
         with pytest.raises(TypeError):
-            parse_url(b"https://www.google.com/")
+            parse_url(b"https://www.google.com/")  # type: ignore[arg-type]
 
     @pytest.mark.parametrize(
         "kwargs, expected",
         [
             pytest.param(
                 {"accept_encoding": True},
                 {"accept-encoding": "gzip,deflate,br"},
-                marks=onlyBrotli(),
+                marks=onlyBrotli(),  # type: ignore[arg-type]
             ),
             pytest.param(
                 {"accept_encoding": True},
                 {"accept-encoding": "gzip,deflate"},
-                marks=notBrotli(),
+                marks=notBrotli(),  # type: ignore[arg-type]
```

Comment:
test/test_util.py:467: error: Argument "marks" to "param" has incompatible type "Callable[[_TestFuncT], _TestFuncT]"; expected "Union[MarkDecorator, Collection[Union[MarkDecorator, Mark]]]"  [arg-type]